cmd/lncli/commands.go

```diff
@@ -2105,3 +2108,41 @@ func updateFees(ctx *cli.Context) error {
 	printRespJSON(resp)
 	return nil
 }
+
+var setAliasCommand = cli.Command{
```

Comment:
Discussed a bit, and it makes sense to only be able to set alias and color at startup, not using RPC. So this command can be removed :)